### PR TITLE
feat(doc, dvclive): add copy parameter description to log_artifact

### PR DESCRIPTION
**Pull request by shcheklein**
Per https://github.com/iterative/dvclive/pull/559

### CODE REVIEW

File: content/docs/dvclive/live/log_artifact.md [hunk 1]
```diff
@@ -36,21 +37,27 @@ with Live() as live:
 
 Uses `dvc add` to track `path` with DVC, generating a `{path}.dvc` file.
 
-When combined with `save_dvc_exp=True`, it will ensure that `{path}.dvc` is
-included in the experiment.
+When combined with [`save_dvc_exp=True`](/doc/dvclive#initialize-dvclive), it
+will ensure that `{path}.dvc` is included in the experiment.
```
Comment (dberenbaum) [inline content/docs/dvclive/live/log_artifact.md:41]: Very minor, but WDYT about combining these paragraphs? It creates a 2 paragraph description that matches what the method does - 1) tracks with dvc and 2) adds artifact to dvc.yaml.

Comment (shcheklein) [reply content/docs/dvclive/live/log_artifact.md:41]: yep, not related to this PR, but I can do it

File: content/docs/dvclive/live/log_artifact.md [hunk 2]
```diff
@@ -36,21 +37,27 @@ with Live() as live:
...
+- `path` - an existing directory or file.
+
+- `name` - an optional custom name of an artifact. If not provided the path stem
+  (last part of the path without the file extension) will be used as the
+  artifact name.
+
+- `copy` - copy a directory or file at `path` into the `dvclive/artifacts`
+  location ([default](/doc/dvclive/how-it-works#directory-structure)) before
+  tracking it. The new path is used instead of the original one to track the
+  artifact.
```
Comment (dberenbaum) [inline content/docs/dvclive/live/log_artifact.md:60]: WDYT about adding something like this? "Useful if you don't want to track the original path in your repo (for example, it is outside the repo or in a Git-ignored directory)."

Comment (dberenbaum) [inline content/docs/dvclive/live/log_artifact.md:60]: @aguschin Sorry I missed this before, but shouldn't we include all the args here? 

Also, do we actually want to support `meta`? Is it used anywhere?

**Review by dberenbaum — APPROVED** [review]
Thanks @shcheklein! Minor comments added but nothing blocking.

File: content/docs/dvclive/live/log_artifact.md (hunk 2 above)
Comment (aguschin) [reply content/docs/dvclive/live/log_artifact.md]: We had a GTO user asking for that field previously. No harm in keeping it I assume?

> Is it used anywhere?

We don't use it. The only way to get it currently is call DVC API or read `dvc.yaml`. In GTO there was a command `gto describe myartifact --custom` which would output content of `custom` field (`meta` is the same field for the same purposes, but with a different name).

Comment (aguschin) [reply content/docs/dvclive/live/log_artifact.md]: > Sorry I missed this before, but shouldn't we include all the args here?

Right, thanks for the notice.

**Pull request merged**